android/src/main/java/com/mapbox/mapboxgl/MapboxMapOptionsSink.java

```diff
@@ -14,6 +14,9 @@
 
 //  void setMapType(int mapType);
 
+  // TODO: styleString is not actually a part of options. consider moving
+  void setStyleString(String styleString); 
```

Comment:
as indicated in comment, options would be a better place for this configuration.